src/Tests/Backend/ONE/ONEToolchain.test.ts

```diff
@@ -65,9 +170,13 @@ suite('Backend', function() {
       });
     });
 
-    // suite('#compiler()', function() {
-
-    // });
+    suite('#compiler()', function() {
+      test('returns toolchainCompiler', function() {
+        const oneBackend = new ONEToolchain();
+        const toolchainCompiler = oneBackend.compiler();
+        assert.instanceOf(toolchainCompiler, ToolchainCompiler);
```

Comment:
We returns this.toolchainCompiler when we executes compiler function. I wonder this is the right way to test this situation.